pkg/webhook/validator_result.go

```diff
@@ -55,6 +55,10 @@ type AuthorityMatch struct {
 // signature could be a signature on the Image (.sig) or on an Attestation
 // (.att).
 type PolicySignature struct {
+	// A unique identifier describing this signature.
+	// This is typically the hash of this signature's OCI layer for images.
+	ID string `json:"id,omitempty"`
```

Comment:
What about adding an ID for the PolicyAttestation too ? It could be the same ID of the PolicySignature it consists of.